contrib/macdeploy/extract-osx-sdk.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+7z x Xcode_7.3.1.dmg 5.hfs
+SDKNAME="MacOSX10.11.sdk"
+SDKDIRINODE=154283
```

Comment:
I didn't see an easy way to do so. We'd have to iterate through each subdirectory's inode manually I think. Either that or go recursive from the root and filter, but that'd be much slower.